google/cloud/spanner_v1/pool.py

```diff
@@ -422,6 +480,18 @@ def bind(self, database):
             session_template=Session(creator_role=self.database_role),
         )
 
+        requested_session_count = request.session_count
```

Comment:
Have you not followed the BatchCreateSession span here?

Reply:
@harshachinta, I did use get_current_span() to retrieve the span down below. This PR is for adding span annotations rather than for creating new spans.